R/misc.R

```diff
@@ -270,10 +270,11 @@ merge_term_info <- function(.new, .old) {
 ellipse_check <- function(...) {
   terms <- quos(...)
   if (is_empty(terms)) {
-    rlang::abort(
-      paste0(
-        "Please supply at least one variable specification.",
-        "See ?selections."
+    cli::cli_abort(
+      c(
+        "!" = "Please supply at least one variable specification.",
+        "i" = "See {.help [?selections](recipes::selections)} \\
+              for more information."
```

Comment:
I didn't add tests for this, as it is technically deprecated. I opened an issue to properly handle that here https://github.com/tidymodels/recipes/issues/1243